alvr/server_openvr/src/props.rs

```diff
@@ -17,11 +17,11 @@ use alvr_session::{
     ControllersEmulationMode, HeadsetEmulationMode, OpenvrPropKey, OpenvrPropType, OpenvrProperty,
 };
 use std::{
-    ffi::{c_char, CString},
+    ffi::{c_char, c_void, CString},
     ptr,
 };
 
-pub fn set_openvr_prop(device_id: u64, prop: OpenvrProperty) {
+pub fn set_openvr_prop(instance_ptr: Option<*mut c_void>, device_id: u64, prop: OpenvrProperty) {
```

Comment:
not the biggest fan of this api, since it behaves like an enum but isn't one, but it's fine if need be

Reply:
What do you mean enum? Do you think we should just check for null pointer? (maybe we should)

Reply:
You use either a device id or a instance ptr, so an enum `DeviceHandle { DeviceId, Pts }`

Reply:
that won't work because I use the device_id always, and only optionally instance_ptr

Reply:
Line 29. it's used later for a debug print line. Not worth to conditionally skip the debug line IMO